locale/de/LC_MESSAGES/pronterface.po

```diff
@@ -587,7 +586,7 @@ msgstr "Keine Verbindung zum Drucker."
 
 #: printrun/printcore.py:609
 msgid "Print start callback failed with:"
-msgstr "Druckstart-Rückmeldung fehlgeschlagen mit:"
+msgstr "Druckstart-Rückruffunktion fehlgeschlagen mit:"
```

Comment:
Can you check if there are other places where "callback" is used? They probably need the same change